airflow/example_dags/example_bash_operator.py

```diff
@@ -17,48 +17,54 @@
 # specific language governing permissions and limitations
 # under the License.
 
-import airflow
 from builtins import range
-from airflow.operators.bash_operator import BashOperator
-from airflow.operators.dummy_operator import DummyOperator
-from airflow.models import DAG
 from datetime import timedelta
 
+import airflow
+from airflow.models import DAG
+from airflow.operators.bash_operator import BashOperator
+from airflow.operators.dummy_operator import DummyOperator
 
-args = {
-    'owner': 'airflow',
-    'start_date': airflow.utils.dates.days_ago(2)
-}
+args = {'owner': 'airflow', 'start_date': airflow.utils.dates.days_ago(2)}
```

Comment:
Don't understand why we need this refactor? 

It looked better formatted.

Reply:
@kaxil will revert back to each pair on separate line.
@KimchaC I didn't refactor the start_date in any of the example DAGs. I imagine dynamic start_date was used to always have the example DAGs start from a recent date instead of a fixed date resulting in a large number of DAG runs when loading the examples.